File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/17.go
package main
import "fmt"
import "strings"

func termina01(s string) bool {
    return strings.HasSuffix(s, "01")
}

func main() {
    fmt.Println(termina01("1101"))  // True
    fmt.Println(termina01("1000"))  // False
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/17.java
public class AFNTermina01 {
    public static boolean termina01(String s) {
        return s.endsWith("01");
    }

    public static void main(String[] args) {
        System.out.println(termina01("1101"));  // True
        System.out.println(termina01("1000"));  // False
    }
}
